imgix/urlbuilder.py

```diff
@@ -93,6 +95,19 @@ def create_url(self, path, params={}, opts={}):
         else:
             domain = self._domains[0]
 
+        regex = re.compile(
```

Comment:
The regex-checking should happen during `UrlBuilder` initialization, since that's where the domain value is actually supplied. Probably right before the assignment on L55.

Reply:
Agreed, I'll make that change.